source/plugin/wgen/modifiers.rst

```diff
@@ -70,8 +70,8 @@ the first argument and your modifier as the second.
 .. code-block:: java
 
     @Listener
-    public void onGameInitialization(GameInitializationEvent event) {
-        Sponge.getRegistry().register(WorldGeneratorModifier.class , new MyModifier());
+    public void onWorldRegistration(GameRegistryEvent.Register<WorldGeneratorModifier> event) {
```

Comment:
I would rather call the method `onWGenModifierRegistration` or `onWorldGeneratorModifierRegistration`.